consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -1685,21 +1673,28 @@ impl<
         // We start on view 1 because the genesis container occupies view 0/height 0.
         self.enter_view(1);
 
+        // Initialize journal
+        let mut journal = Journal::<_, _, Voter<C::Signature, D>>::init(
+            self.context.with_label("journal"),
+            JConfig {
+                partition: self.partition.clone(),
+                compression: self.compression,
+                codec_config: usize::MAX, // anything we read from journal is already verified
```

Comment:
This will be cleaned up with: https://github.com/commonwarexyz/monorepo/issues/815